src/components/user-settings.tsx

```diff
@@ -72,13 +74,11 @@ const SnapshotInfo = ( {
 	const isOffline = useOffline();
 	const offlineMessage = __( 'Deleting demo sites requires an internet connection.' );
 	return (
-		<div className="flex gap-5 flex-col">
-			<h2 className="a8c-subtitle-small">{ __( 'Usage' ) }</h2>
-			<div className="flex gap-4 flex-row items-center w-full">
+		<div className="flex gap-3 flex-col">
+			<h2 className="a8c-label-semibold">{ __( 'Demo Sites' ) }</h2>
```

Comment:
```suggestion
			<h2 className="a8c-label-semibold">{ __( 'Demo sites' ) }</h2>
```

It seems to be in the lower case in the screenshot on https://github.com/Automattic/dotcom-forge/issues/7409

Reply:
Updated! Thanks!